api/src/main/java/org/pmiops/workbench/opsgenie/OpsGenieService.java

```diff
@@ -0,0 +1,9 @@
+package org.pmiops.workbench.opsgenie;
+
+import org.pmiops.workbench.model.EgressEvent;
+
+public interface OpsGenieService {
+
+  // Creates an Opsgenie alert for a high-egress event detected in the Workbench system.
+  public void createEgressEventAlert(EgressEvent egressEvent);
+}
```

Comment:
It would be a YAGNI smell – for now, there is no active flow where the Workbench should close a high-egress alert. It can only open a new alert or add a note to an existing alert.

I could imagine such a thing being useful for an end-to-end test of the high-egress alerting functionality; it will be trivial to add at that point.